benches/benches/bevy_ecs/world/world_get.rs

```diff
@@ -51,9 +51,10 @@ pub fn world_entity(criterion: &mut Criterion) {
     for entity_count in RANGE.map(|i| i * 10_000) {
         group.bench_function(format!("{entity_count}_entities"), |bencher| {
             let world = setup::<Table>(entity_count);
+            let offset = world.resource_count();
```

Comment:
This is so fragile: I *really* don't like this.

Reply:
This is exactly correct. I've justified it to myself by noting that creating entities from raw numbers *is* really fragile, so this is not really *that* bad in comparison.

It's not great either.

Reply:
`setup` calls `world.spawn_batch`, which returns an iterator that yields the actual `Entity` values.  Would it work to stick those in a `Vec<Entity>` and iterate *that* instead of using integers and `Entity::from_raw`?